CondCore/BeamSpotPlugins/interface/BeamSpotPayloadInspectorHelper.h

```diff
@@ -681,14 +685,15 @@ namespace simBeamSpotPI {
    */
   template <class PayloadType>
   class SimBSParamsHelper {
-    typedef std::array<double, 9> bshelpdata;
+    typedef std::array<double, 10> bshelpdata;
 
   public:
     SimBSParamsHelper(const std::shared_ptr<PayloadType>& bs) {
       // fill in the values
       m_values[0] = bs->x(), m_values[1] = bs->y(), m_values[2] = bs->z();
       m_values[3] = bs->sigmaZ(), m_values[4] = bs->betaStar(), m_values[5] = bs->emittance();
-      m_values[6] = bs->phi(), m_values[7] = bs->alpha(), m_values[8] = bs->timeOffset();
+      m_values[6] = (1 / std::sqrt(2)) * std::sqrt(bs->emittance() * bs->betaStar()) * 10000.f;
+      m_values[7] = bs->phi(), m_values[8] = bs->alpha(), m_values[9] = bs->timeOffset();
```

Comment:
```suggestion
      m_values[parameters::expTransWidth] = (1 / std::sqrt(2)) * std::sqrt(bs->emittance() * bs->betaStar()) * 10000.f;
      m_values[parameters::phi] = bs->phi(), m_values[parameters::alpha] = bs->alpha(),
      m_values[parameters::timeOffset] = bs->timeOffset();
```

Reply:
thanks, addressed in [9877ac0](https://github.com/cms-sw/cmssw/pull/41967/commits/9877ac09b974681ef5d0bf058326d23e5f0bc8e0)